docs_src/getting-started/Ch-GettingStartedHybrid.md

```diff
@@ -1,156 +1,80 @@
-
-[Clients]
-[Clients.Data]
-Name = "edgex-core-data"
-Protocol = "http"
-Host = "172.17.0.1"
-Port = 48080
-Timeout = 5000
-
-[Clients.Metadata]
-Name = "edgex-core-metadata"
-Protocol = "http"
-Host = "172.17.0.1"
-Port = 48081
-Timeout = 5000
-
-[Clients.Logging]
-Name = "edgex-support-logging"
-Protocol = "http"
-Host = "172.17.0.1"
-Port = 48061
-```
-
-Save the configuration.toml file after making these changes.
-
-**Run device-virtual**
-
-Now you can run device-virtual-go in Go Land that uses the rest of EdgeX
-Dockerized services. For example, in the Go Land terminal, make sure you
-are in the device-virtual-go directory and first build the service with
-a call to make build. After successfully building the service, run the
-service by going into the cmd folder and executing ./device-virtual (as
-shown below).
-
-![image](EdgeX_GettingStartedHybridGoLand.png)
-
-**Check the Results**
-
-At this time, your virtual device micro service running in Go Land
-should be communicating with the other EdgeX micro services running in
-their Docker containers. Give the virtual device a few seconds or so to
-initialize itself and start sending data to Core Data. To check that it
-is working properly, open a browser and point your browser to Core Data
-to check that events are being deposited. You can do this by calling on
-the Core Data API that checks the count of events in Core Data
-(<http://[host].48080/api/v1/event/count>).
-
-![image](EdgeX_GettingStartedHybridResults.png)
+    Consul, Redis, Core Data, Core Metadata, Support Notifications, and Core Command. 
+
+    Based on the instructions found in the [Getting Started with Docker](Ch-GettingStartedUsers.md#Get-Run-EdgeX-Foundry), locate and download the appropriate Docker Compose file for your development environment.  Next, issue the following commands to start this set of EdgeX containers - providing a minimal functioning EdgeX environment. 
+    ``` bash
+    docker-compose up -d consul
+    docker-compose up -d redis
+    docker-compose up -d notifications
+    docker-compose up -d metadata
+    docker-compose up -d data
+    docker-compose up -d command
+
+    ```
+
+    !!! Note
+        These notes assme you are working with the EdgeX Genva release.  Some versions of EdgeX may require other or additional containers to run.
+    
+3.  Run **docker-compose ps** to confirm that all the
+containers have started.
+
+## Get, Build and Run the Service
+With the EdgeX containers running, you can now download, build and run natively (outside of a container) the service you want to work on.  In this example, the virtual device service is used to exemplify the steps necessary to get, build and run the native service with the EdgeX containerized services.  However, the practice could be applied to any service.
+
+1.  Get the service code
+    Per [Getting Started Go Developers](./Ch-GettingStartedGoDevelopers.md#Get-the-code), pull the micro service code you want to work on from GitHub. In
```

Comment:
If you are meaning for this to be on its own line, then you need another blank line - currently it renders as a run on sentence. 

Giving it some thought -- i think these would be better as headers so they show up and are navigable on the TOC on the right which is kind of bare as it stands.

Reply:
agree